bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -96,6 +95,9 @@ spec:
         - name: volume-permissions
           image: {{ include "mongodb.volumePermissions.image" . }}
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" (or .Values.volumePermissions.image.debug .Values.diagnosticMode.enabled) | quote }}
```

Comment:
`volumePermissions` container does not have `BITNAMI_DEBUG` because it does not have a bitnami entrypoint

Reply:
you right!